### PR TITLE
Add CI

### LINKED ISSUE #19: Add CI
Add CI with the following stages:
* build
* testing
* coverage (with enforced and configurable threshold)
* linting

### PR DESCRIPTION
**Pull request by thomas-fossati**
Fixes #19

### CODE REVIEW

File: .github/workflows/ci-go-cover.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Copyright 2020-present Montgomery Edwards⁴⁴⁸ (github.com/x448).
```
Comment (yogeshbdeshpande) [inline .github/workflows/ci-go-cover.yml:1]: Please change the copyright year to 2021

File: .github/workflows/ci-go-cover.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+# This file is licensed under the MIT License. See LICENSE at https://github.com/x448/workflows for the full text.
+#
+# CI Go Cover 2020.1.28.
```
Comment (yogeshbdeshpande) [inline .github/workflows/ci-go-cover.yml:4]: Please revisit the date

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# GitHub Actions - CI for Go to build & test.  See ci-go-cover.yml and linters.yml for code coverage and linters.
```
Comment (yogeshbdeshpande) [inline .github/workflows/ci.yml:1]: Just a questions, are these new files does not need a Copyright header like cover.yml?

File: scripts/cov.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import os
+import re
+import sys
+
+# read output of (potentially many invocations of) "go test -cover -short",
+# e.g., "coverage: 65.7% of statements"
+cover_report_lines = sys.stdin.read()
+
+if len(cover_report_lines) == 0:
```
Comment (yogeshbdeshpande) [inline scripts/cov.py:9]: is there a possibility to print some error condition here ? how does the output know of this specific failure?

File: .github/workflows/ci-go-cover.yml (hunk 1 above)
Comment (thomas-fossati) [reply .github/workflows/ci-go-cover.yml:1]: the author (x448) did the original thing that I then modified in 2020, I am not authoritative in this matter :-)

File: .github/workflows/ci-go-cover.yml (hunk 2 above)
Comment (thomas-fossati) [reply .github/workflows/ci-go-cover.yml:4]: ditto

File: .github/workflows/ci.yml (hunk 3 above)
Comment (thomas-fossati) [reply .github/workflows/ci.yml:1]: This is stuff I've stolen from fxamacker/cbor, it didn't have a header in the original so I didn't dare adding anything.

File: scripts/cov.py (hunk 4 above)
Comment (thomas-fossati) [reply scripts/cov.py:9]: This script outputs nothing, it returns an exit code.  If it's important to differentiate we could exit with a different code.

File: scripts/cov.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    sys.exit(1)
```
Comment (thomas-fossati) [inline scripts/cov.py:10]:
```suggestion
    sys.exit(2)
```

File: scripts/cov.py (hunk 4 above)
Comment (thomas-fossati) [reply scripts/cov.py:9]: Done in 1faca96

**Pull request merged**